components/01-atoms/text/text/07-byline.twig

```diff
@@ -0,0 +1,27 @@
+{#
+/**
+ * Available variables:
+ * - byline_content - the content of the byline (typically text)
```

Comment:
It seems this variables in comments need to be updated to match with the variables